### PR TITLE
FEATURE: atDude Tutorial Steps 4, 5 & 6

### PR DESCRIPTION
**Pull request by CurtlyCritchlow**
**- What I did**
created steps 4, 5 and 6 to the atDude tutorial series. It is still in draft, I would like  someone to review it and change draft to true. I also made minor edits to previous tutorials.

I also created the source code for the tutorials available [here](https://github.com/atsign-foundation/at_demos)
**- How I did it**
Created the folders with their respective index.md and assets files.
**- How to verify it**
Look at the index.md file in subfolders [here](https://github.com/atsign-foundation/docs.atsign.com/tree/trunk/content/docs/tutorials/at-dude)

**- Description for the changelog**

Added steps 4, 5 and 6 to the atDude tutorial series.

### CODE REVIEW

**Review by TylerTrott — DISMISSED** [review]
Woohoo! Amazing @CurtlyCritchlow ! Don't forget to use semantic pr's now as well. Will wait for approval from @XavierChanth before merge

File: content/docs/tutorials/at-dude/_index.md
Comment (JeremyTubongbanua) [inline content/docs/tutorials/at-dude/_index.md:5]: *"Learn how to build **a** production app on the atPlatform" ?

File: content/docs/tutorials/at-dude/5-reset-atsign/index.md
Comment (JeremyTubongbanua) [inline content/docs/tutorials/at-dude/5-reset-atsign/index.md:7]: If this is finished, then set it to: `draft: false`

File: content/docs/tutorials/at-dude/6-send_dude-screen-app-bar/index.md
Comment (JeremyTubongbanua) [inline content/docs/tutorials/at-dude/6-send_dude-screen-app-bar/index.md:7]: Same thing here, `draft: false`
If you want to merge it as `draft: true`, then just ignore this review. =)

**Review by JeremyTubongbanua — DISMISSED** [review]
@CurtlyCritchlow small changes, but LGTM & great work :)

File: content/docs/tutorials/at-dude/_index.md
Comment (TylerTrott) [reply content/docs/tutorials/at-dude/_index.md:5]: @CurtlyCritchlow

File: content/docs/tutorials/at-dude/5-reset-atsign/index.md
Comment (TylerTrott) [reply content/docs/tutorials/at-dude/5-reset-atsign/index.md:7]: @CurtlyCritchlow

File: content/docs/tutorials/at-dude/6-send_dude-screen-app-bar/index.md
Comment (TylerTrott) [reply content/docs/tutorials/at-dude/6-send_dude-screen-app-bar/index.md:7]: @CurtlyCritchlow

**Comment by cpswan** [comment]
@CurtlyCritchlow it looks like this got to LGTM, and just needed the draft status to be fixed before it could be merged. I know that there's now redesign stuff in the works, but unless the changes driven by that are really quick I think it's better to get this published and change it later.

**Comment by xavierchanth** [comment]
@cpswan I will merge this so we can see what it looks like in staging, then we can set this to `draft: false` and do a release.

**Review by xavierchanth — APPROVED** [review]

**Pull request merged**
